composable-architecture-test/src/main/kotlin/composablearchitecture/test/TestStore.kt

```diff
@@ -70,6 +70,20 @@ private constructor(
                 Prism.id(),
                 testDispatcher
             )
+        fun <State, Action : Comparable<Action>, Environment> create (
+            state: State,
+            reducer: Reducer<State, Action, Environment>,
+            environment: Environment,
+            testDispatcher: TestCoroutineDispatcher = TestCoroutineDispatcher()
+        ) =
+            TestStore(
+                state,
+                reducer,
+                environment,
+                Lens.id(),
+                Prism.id(),
+                testDispatcher
+            )
```

Comment:
What would be the benefit of introducing `create` over the already existing `invoke` method? You should be able to instantiate it with `TestStore(...)` vs `TestStore.create(...)`.

Reply:
When you mess up something while instantiating a test store (types or something), using the `invoke` the error message is not clear (or non-existent). This was more like for debugging, but I can take it out.

Reply:
I would keep the API surface to a minimum for now and introduce new ones only if necessary. If we have concrete examples and better understand what's happening with `invoke`, we can reconsider `create` (or any alternatives).